staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -450,6 +450,13 @@ type PersistentVolumeClaimSpec struct {
 	// More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#resources
 	// +optional
 	Resources ResourceRequirements `json:"resources,omitempty" protobuf:"bytes,2,opt,name=resources"`
+	// AllocatedResources represents the resources that was requested to satisfy user
+	// request. This field is not meant to be set by the user but is used for tracking
```

Comment:
what happens if a user does set it?  Is there protection that prevents a user from setting this (I didn't see it right off).  My memory is that users can set fields on PVC.spec.

Reply:
if user does set it then - https://github.com/kubernetes/kubernetes/pull/92770/files#diff-88702b41a33bc03bef189034a3c7e7f6R45 will replace/overwrite it with higest observed requested resource.